llpc/lower/llpcSpirvLowerGlobal.cpp

```diff
@@ -1780,7 +1780,9 @@ void SpirvLowerGlobal::storeOutputMember(Type *outputTy, Type *storeTy, Value *s
 // =====================================================================================================================
 // Find a new insertion point that dominates the current insertion point \p instA and a specified instruction use
 // \p useB.
-static Instruction *findNearestCommonDominator(Instruction *instA, Use &useB, DominatorTree &domTree) {
+static Instruction *findNearestCommonDominator(Instruction *instA, Use &useB, DominatorTree &domTree,
+                                               SmallVector<Instruction *, 8> &usesToReplace,
```

Comment:
Hmm, wouldn't it be better to not pass these vectors here, and instead just make findNearestCommonDominator tell the caller that useB is not in the dom tree by returning nullptr or something?

Does that contradict what Jay said?

Also, if you are going to pass vectors here, use SmallVectorImpl so the function def/decl doesn't need to know what default capacity you used in the vector.

If you do go with this approach, the comment above the function is inadequate.

Reply:
Going with Ruiling's suggestion.